cli/fs.js

```diff
@@ -4,6 +4,10 @@ import { dirname, resolve } from "node:path";
 import * as yaml from "js-yaml";
 import * as ailly from "@ailly/core";
 
+/**
+ * @param {ReturnType<import("./args.js").makeArgs>} args
+ * @returns
```

Comment:
This will break the type inference for the actual return type.

Reply:
Done